pkgs/applications/terminal-emulators/syncterm/default.nix

```diff
@@ -8,7 +8,15 @@ stdenv.mkDerivation rec {
     url = "mirror://sourceforge/${pname}/${pname}-${version}-src.tgz";
     sha256 = "19m76bisipp1h3bc8mbq83b851rx3lbysxb0azpbr5nbqr2f8xyi";
   };
-  sourceRoot = "${pname}-${version}/src/syncterm";
+
+  patches = [
+    # Cherry-picks from the upstream Synchronet tree, removing calls to `pthread_yield`.
+    # See upstream issue: https://gitlab.synchro.net/main/sbbs/-/issues/299
+    ./0001-deprecate-pthread-yield-851627df.patch
```

Comment:
This one applies cleanly, so it would be good to `fetchpatch` it.  The other one does not, so it's fine to include that patch in Nixpkgs.

Reply:
Done!